qi-lib/flow/core/passes/pass-1000-qi0.rkt

```diff
@@ -0,0 +1,423 @@
+#lang racket/base
+
+(require "../passes.rkt"
+         (prefix-in fancy: fancy-app)
+         "../impl.rkt"
+         racket/function
+         (only-in racket/list make-list)
+         (for-syntax racket/base
+                     syntax/parse
+                     "../syntax.rkt"
+                     "../../aux-syntax.rkt"
+                     (only-in racket/list make-list)
+                     ))
+
+(begin-for-syntax
+  
+  (define-and-register-pass 1000 (qi0-wrapper stx)
+    (syntax-parse stx
+      (ex #'(qi0->racket ex))))
+
+  )
+
+(define-syntax (qi0->racket stx)
```

Comment:
This is mostly from qi-lib/flow/core/compiler.rkt, right? I think I was
expecting to see the corresponding code deleted from there.

Reply:
Yes, it has been deleted.